src/components/shared/accountMigration/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import CopyToClipboard from '@toolbox/copyToClipboard';
+import Icon from '@toolbox/icon';
+import LiskAmount from '@shared/liskAmount';
+import AccountVisualWithAddress from '@shared/accountVisualWithAddress';
+import styles from './index.css';
+
+const AccountMigration = ({ t, account, showBalance }) => (
```

Comment:
If this component does not use other data from `account`, please only send in the `address`

Reply:
It uses balance from both legacy and summary.